buildSrc/subprojects/configuration/src/main/kotlin/org/gradle/gradlebuild/java/AvailableJavaInstallations.kt

```diff
@@ -106,8 +106,8 @@ open class AvailableJavaInstallations(private val project: Project, private val
     private
     fun validateProductionJdks(): Map<String, Boolean> =
         mapOf(
-            "Must use Oracle JDK 9 to perform this build. Is currently ${currentJavaInstallation.vendorAndMajorVersion} at ${currentJavaInstallation.javaHome}." to
-                (currentJavaInstallation.vendorAndMajorVersion != oracleJdk9)
+            "Must use OpenJDK 11 to perform this build. Is currently ${currentJavaInstallation.vendorAndMajorVersion} at ${currentJavaInstallation.javaHome}." to
```

Comment:
Maybe we can simplify this so next time we update we don't have to change several places in the code.
- Name the const `productionJDKName`
- Use the const in the error message `Must use $productionJDKName to...`

Reply:
Thanks, this makes perfect sense.